lib/Eclair/Error.hs

```diff
@@ -140,7 +140,7 @@ wildcardInAssignmentToReport file fileContent spanMap (WildcardInAssignment assi
 
 -- NOTE: pattern match is done this way to keep track of additional errors that need to be reported
 semanticErrorsToReports :: FilePath -> Text -> SpanMap -> SemanticErrors -> [Report Text]
-semanticErrorsToReports file fileContent spanMap e@(SemanticErrors _ _ _ _ _ _) =
+semanticErrorsToReports file fileContent spanMap e@(SemanticErrors {}) =
```

Comment:
Actually I don't want this, on purpose. There's a chance I forget to add the new error to the `concat` when I add it to the `SemanticErrors` data type.

Can we turn this specific warning off (only for this module)?